src/views/acquirer/created-request-outcome.njk

```diff
@@ -0,0 +1,52 @@
+{% extends "page.njk" %}
+
+{% if request.sharedata.status === "ACCEPTED" %}
+  {% set nextSteps = "Your department's contact email has been shared with the " + request.assetPublisher.title + " so they can begin working with you on the data share arrangement. You are able to contact them at data-management@test.co.uk" %}
+{% else %}
+  {% set nextSteps = "This request has been rejected. Consider the feedback before making further data share requests." %}
+{% endif %}
+
+{% block content %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-l">Request ID: {{request.sharedata.requestId}}</span>
+    <h1 class="govuk-heading-xl">
+      {{ request.sharedata.status | capitalize }} request
```

Comment:
The `request` itself also has a status, which is more likely to be kept up to date than the status in the sharedata. If we have time in the future I'd like to refactor this all so that `sharedata` *only* has the request form answers, and all the status, IDs etc. are only stored in our database.
(We didn't have the db set up when we started this, which is why everything's ended up in the sharedata)

```suggestion
      {{ request.status | capitalize }} request
```

Reply:
Thanks Kiran, I've made the suggested changes 😊